### DISCUSSION TITLE
关于chatglm3 agent 能力

### DISCUSSION THREAD

**Discussion by shell-nlp** [zai-org/ChatGLM3#854] [Q&A] [answered]
agent 中，langchain_demo 和 openai_demo 的提示词是不一样的， 它们有什么区别

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
langchain_demo 的提示词才能接入到langchain中，这是跟框架有关系的，lc要用json返回，如果用关键词返回的话会无法捕获
